css-layout-classes/styles.css

```diff
@@ -0,0 +1,28 @@
+body {
+  font-family: sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-one-fourth {
+  flex-basis: calc(100% / 4);
+}
+
+.four {
+  max-width: 100%;
```

Comment:
max width is a limiting property, not a set property. you want the images to be `100%` of the width of their parents, and this needs to apply to all images.